src/dosbox.cpp

```diff
@@ -602,28 +603,35 @@ void DOSBOX_Init()
-	        "                 a unique Contrast control that dims non-bright colors only\n"
-	        "                 (0 to 100; defaults to 100).\n"
+	        "Sets the interpretation of CGA RGBI colors. Affects all machine types capable\n"
+	        "of displaying CGA or better graphics. Built-in presets:\n"
+	        "  default:       The canonical CGA palette, as emulated by VGA adapters\n"
+	        "                 (default).\n"
+	        "  tandy [BL]:    Emulation of an idealised Tandy monitor with adjustable\n"
+	        "                 Brown Level (0 - red, 50 - brown, 100 - dark yellow;\n"
+	        "                 defaults to 50).\n"
+	        "  tandy-warm:    Emulation of the actual color output of an unknown Tandy\n"
+	        "                 monitor.\n"
+	        "  ibm5153 [C]:   Emulation of the actual color output of an IBM 5153 monitor\n"
+	        "                 with a unique Contrast control that dims non-bright colors\n"
+	        "                 only (0 to 100; defaults to 100).\n"
 	        "  agi-amiga-v1, agi-amiga-v2, agi-amiga-v3:\n"
 	        "                 Palettes used by the Amiga ports of Sierra AGI games\n"
 	        "                 (see the manual for further details).\n"
-	        "  agi-amigaish:  A mix of EGA and Amiga colors used by the Sarien AGI-interpreter.\n"
+	        "  agi-amigaish:  A mix of EGA and Amiga colors used by the Sarien\n"
+	        "                 AGI-interpreter.\n"
 	        "  scumm-amiga:   Palette used by the Amiga ports of LucasArts EGA games.\n"
 	        "  colodore:      Commodore 64 inspired colors based on the Colodore palette.\n"
 	        "  colodore-sat:  Colodore palette with 20% more saturation.\n"
-	        "  dga16:         A modern take on the canonical CGA palette with dialed back contrast.\n"
-	        "You can also set custom colors by specifying 16 space or comma separated color values,\n"
-	        "either as 3 or 6-digit hex codes (e.g. #f00 or #ff0000 for full red), or decimal\n"
-	        "RGB triplets (e.g. (255, 0, 255) for magenta). The 16 colors are ordered as follows:\n"
-	        "black, blue, green, cyan, red, magenta, brown, light-grey, dark-grey, light-blue,\n"
-	        "light-green, light-cyan, light-red, light-magenta, yellow, and white.\n"
+	        "  dga16:         A modern take on the canonical CGA palette with dialed back\n"
+	        "                 contrast.\n"
+	        "You can also set custom colors by specifying 16 space or comma separated color\n"
+	        "values, either as 3 or 6-digit hex codes (e.g. #f00 or #ff0000 for full red),\n"
+	        "or decimal RGB triplets (e.g. (255, 0, 255) for magenta). The 16 colors are\n"
+	        "ordered as follows:\n"
+	        "black, blue, green, cyan, red, magenta, brown, light-grey, dark-grey,\n"
+	        "light-blue, light-green, light-cyan, light-red, light-magenta, yellow,\n"
+	        "and white.\n"
```

Comment:
Not sure if it's still possible (or if we care), but I tried to arrange the labels with the values 1:1, so users can visually connect them (if editing from a list):

![2022-10-04_12-59](https://user-images.githubusercontent.com/1557255/193914013-f8c73b65-a7a3-4921-af1e-afb01ab20701.png)

Reply:
I did a small change so they are easier to connect now - the only exception is _white_. Of cource free to re-open conversation if you have a better idea.

Reply:
Perfect; and very easy to eyeball each value (the trailing white no problem :+1: )